index.html

```diff
@@ -1,7 +1,30 @@
----
-layout: default
-description: "Learn how to make your first web app with Ruby on Rails! Sign up for a Rails Girls workshop in your area and follow these guides."
----
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

Comment:
Why do you introduce these additional tags? It's also wrong.

Reply:
okay sorry for that